build/test-replacements.js

```diff
@@ -106,6 +112,18 @@ module.exports['common.js'] = [
         + '}\n'
         + '/*</replacement>*/\n'
     ]
+  , [
+        /^if \(global\.ArrayBuffer\) \{([^\}]+)\}$/m
+      ,   '/*<replacement>*/if (!process.browser) {'
+        + '\nif \(global\.ArrayBuffer\) {$1}\n'
+        + '}/*</replacement>*/\n'
+    ]
+    , [
+          /^Object\.defineProperty\(([\w\W]+?)\}\)\;/mg
```

Comment:
same here, this defines some stuff we never actually use and causes issues in older browsers that don't have a working copy of Object.defineProperty